lib/GlobalStylesheets.js

```diff
@@ -30,13 +30,66 @@ function addStyle(css){
 
 function createStylesheet(stylesheet) {
   var style = assign({}, stylesheet.style);
+  var hoverStyles = {};
+  var activeStyles = {};
+  var focusStyles = {};
+
+  Object.keys(style).forEach(function(key) {
```

Comment:
Please use a `for in` loop here to iterate over the keys so we don't allocate an extra array